axlearn/common/inference_test.py

```diff
@@ -292,17 +295,14 @@ def init_state(prng_key):
             return state, ckpt_dir
 
     @parameterized.parameters(
-        filter(
-            lambda params: is_supported(*params),
-            itertools.product(
-                ("cpu", "gpu", "tpu"),  # platform,
-                ((1, 1), (4, 1), (2, 2), (8, 1), (4, 2)),  # mesh_shape
-                (jnp.float32, jnp.bfloat16),  # param_dtype
-                (None, jnp.float32, jnp.bfloat16),  # inference_dtype
-                (1, 16),  # global_batch_size
-                (DataPartitionType.FULL, DataPartitionType.REPLICATED),  # data_partition
-                (True, False),  # whether use ema weight
-            ),
+        itertools.product(
```

Comment:
instead of filtering in @parameterized.parameters (which means tests are not skipped but disappear and can result in pytest error) I'm doing pytest.skip in the function (line 65)